pom.xml

```diff
@@ -337,6 +337,13 @@
       <version>${powermock.version}</version>
       <scope>test</scope>
     </dependency>
+    <!-- till https://github.com/checkstyle/checkstyle/issues/7368 -->
```

Comment:
This solves [BAD LDC: 17](https://github.com/jboss-javassist/javassist/issues/282) issue.